### ISSUE TITLE
No input for outdirectory

### ISSUE THREAD

**Issue by Rahuljagwani** [ControlCore-Project/concore-editor#132]
@pradeeban 
As concore provides ability to give output in separate output directory with command:
./makestudy example/demo <outDirName>
In backend, build method also contains feature to take output Directory Name.
Therefore, there must be an optional input given by user in this regard.

If this issue is valid assign me this.

**Comment by pradeeban** [comment]
Yes, this is valid. We did not have this implemented because there was no integration of concore and concore-editor (until you attempted to it recently).
